denizen_scripts/survival/repo-link/command_heal.dsc

```diff
@@ -12,8 +10,11 @@ heal_command:
     - if <context.args.is_empty>:
         - heal <player>
         - feed 20 <player>
+        - narrate "You have been healed"
     - else:
         - define user <context.args.first>
         - inject player_verification
         - heal <[user]>
         - feed 20 <[user]>
```

Comment:
The syntax for this command is `feed (<entity>) (amount:<#>) (saturation:<#.#>)`
```suggestion
        - feed amount:20 <[user]>
```